src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeZoneToStringTransformer.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+
+/**
+ * Transforms between a timezone identifier string and a DateTimeZone object.
+ *
+ * @author Roland Franssen <franssen.roland@gmai.com>
+ */
+class DateTimeZoneToStringTransformer implements DataTransformerInterface
+{
+    private $multiple;
+
+    public function __construct($multiple = false)
+    {
+        $this->multiple = $multiple;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function transform($dateTimeZone)
+    {
+        if (null === $dateTimeZone) {
+            return;
+        }
+
+        if ($this->multiple) {
+            if (!is_array($dateTimeZone)) {
+                throw new TransformationFailedException('Expected an array.');
+            }
+
+            $transformer = new self();
+
+            return array_map(function ($dateTimeZone) use ($transformer) {
+                return $transformer->transform($dateTimeZone);
+            }, $dateTimeZone);
```

Comment:
minor simplification `return array_map(array($transformer, 'transform'), $dateTimeZone);`? however, the actual is fine to me ;)